Configuration/PyReleaseValidation/python/relval_2026.py

```diff
@@ -27,6 +27,7 @@
 numWFIB.extend([24834.0,24834.911,24834.103]) #2026D98 DDD XML, DD4hep XML, aging
 numWFIB.extend([25061.97]) #2026D98 premixing stage1 (NuGun+PU)
 numWFIB.extend([24834.5,24834.9]) #2026D98 pixelTrackingOnly, vector hits
+numWFIB.extend([24834.72026]) #2026D98 mkFit tracking (initialStep)
```

Comment:
can this be just .7?
"2026" in 72026 is a bit of a dead end (can't be similarly usefully extended to add slight similar/modified; unclear if it makes it easier to remember).
The neighboring tracking wf is `.701`; just the next one will be `.702`

Reply:
> can this be just .7?

I was looking at the implementation and it seems like having conditionals inside of the old workflow is not possible. 
OTOH, it's not obvious that two workflows with the same offset value is impossible

Reply:
`.7` is identified as duplicate. Changed to `.702`